### ISSUE TITLE
upgrade react-scripts

### ISSUE THREAD

**Issue by alyshae** [weibenfalk/react-snake-starter-files#1]
I had to update react-scripts to 3.4.0 in order to get this running. Just thought you might want to know. Thanks for the tutorial.

More details: https://github.com/facebook/create-react-app/issues/8490

**Comment by felixrichnow** [comment]
Yes, I had the exact same issue. I'll write him a comment and tell him about it too @alyshae

**Comment by weibenfalk** [comment]
Thanks for you comments. Really strange that you had to update the scripts. I created this game on 3.3.0 and just tried it locally also and it works for me. =) But I'll update the scripts. Thanks for the notification.
